packages/@okta/vuepress-theme-prose/assets/css/abstracts/_variables.scss

```diff
@@ -1,5 +1,23 @@
 $container-width: 1340px;
 $container-width-desktop-large: 1600px;
+$base-font-size: 16px;
+$scale-ratio: 1.2308;
+$base-line-height: 1.5;
+$heading-line-height: 1.2;
+$max-line-length: 32em;
+
+// Spacing Variables
+
+$base-spacing: $base-line-height * 1rem;
+$small-spacing: $base-spacing / 2;
+$tiny-spacing: $small-spacing / 2;
+$big-spacing: $base-spacing * 2;
+$huge-spacing: $big-spacing * 2;
+
+$em-base-spacing: $base-line-height * 1em;
+$em-small-spacing: $em-base-spacing / 2;
+$em-tiny-spacing: $em-small-spacing / 2;
+$em-big-spacing: $em-base-spacing * 2;
```

Comment:
nit: check for unused variables. I can't find any usage for `em-xxx` - we can drop this section completely. We only use `$base-spacing` and `$small-spacing` idk if we want to keep the other three.
`$scale-ratio` and `$heading-line-height` are also unused

Reply:
I'll remove everything except `$scale-ratio` since we are overriding its value coming from odyssey package here (in odyssey its 1.1487). It might end up breaking something that's being consumed directly from odyssey package.

Reply:
Done.